apps/web/src/store/modules/user/actions.ts

```diff
@@ -131,7 +149,10 @@ export const signIn = async ({ commit }, signInRequest: SignInRequest): Promise<
 export const signOut = (): void => {
     SpaceConnector.flushToken();
 };
-
+const getRoleTypeFromToken = (token: string): RoleType => {
+    const decodedToken = jwtDecode<JWTPayload>(token);
+    return decodedToken.rol;
```

Comment:
question: Isn't it 'rol' instead of 'role'?

Reply:
Yes, It isn't.
`rol` is a JWT Token internal specification.

Reply:
Backend spec..